kubernetes_workflows/retool-code-executor-container.yaml

```diff
@@ -0,0 +1,42 @@
+piVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    retoolService: code-executor
+  name: code-executor
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      retoolService: code-executor
+  revisionHistoryLimit: 3
+  template:
+    metadata:
+      labels:
+        retoolService: code-executor
+    spec:
+      containers:
+        - args:
+            - bash
+            - start.sh
+          env:
+            - name: NODE_ENV
+              value: production
+            - name: NODE_OPTIONS
+              value: --max_old_space_size=1024
```

Comment:
I think there was a bug with setting `NODE_OPTIONS` envvar and running code executor via pkg? but @demitrin would know more if it may apply here

Reply:
code executor doesnt use pkg. unclear if we should set the max_old_space_size tho. i think that's like garbage collection limits? i suspect we need to set it higher in some cases